src/pages/login/login.js

```diff
@@ -76,9 +76,7 @@ const Login = () => {
                       data-cy='email'
                       id='email'
                       label={t('login.placeholders.email')}
-                      className={`${styles.emailInput} ${
-                        errors.email === t('error.profile.email') && styles.afterText
-                      }`}
+                      className={`${styles.emailInput} ${errors.email && styles.afterText}`}
```

Comment:
if there are no email errors then we'll get undefined string as result. It's possible?
![image](https://user-images.githubusercontent.com/93188288/140540668-4cda9bce-ac5d-4617-849b-477d4c3eb8c0.png)

Reply:
You are right. I have fixed it.